pixart.js

```diff
@@ -1 +1,27 @@
+var setColorButton = $('#set-color');
+var inputSelector = $('#color-field');
+var swathDiv = $('.brush');
+var drawColor=null;
 
+swathDiv.on('click',function(){
+  drawColor=$(this).css('backgroundColor');
+});
+
+var cycle = 0;
+var arr = $('.brush');
+setColorButton.on('click', function(evt){
+  evt.preventDefault();
+  arr.eq(cycle).css('backgroundColor',inputSelector.val());
+  cycle++;
```

Comment:
very interesting implementation! again, I recommend thinking of a solution without using a global variable to keep track of the index